### PR TITLE
#Issue:Confusing naming of Flask endpoints solution

### PR DESCRIPTION
**Pull request by raftar2097**

### CODE REVIEW

**Comment by SnaKyEyeS** [comment]
Thanks for the contribution ! However, you also need to change the methods name in the `calendar.js` file for this to work.

**Comment by jeertmans** [comment]
Yes and the tests in `/tests/view/test_calendar.py` should also be updated then :) @raftar2097

**Comment by raftar2097** [comment]
@jeertmans  done that for test also

**Comment by raftar2097** [comment]
@jeertmans  anything more needs to be done

**Comment by jeertmans** [comment]
@raftar2097 yes, as pointed out by @SnaKyEyeS , the file `static/js/calendar.js` has to be edited the same way, to reflect the changes you did in the Python files. After that I think it will be done, thanks !

File: tests/views/test_calendar.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def test_update_custom_event(client, user):
     param = {**changes, **dict(schedule_number=0, color="Color")}
 
     # Test non-recurring event
-    rv = client.post(
+    rv = client.patch(
```
Comment (jeertmans) [inline tests/views/test_calendar.py:350]: Actually you have mistaken `calendar.update_custom_event` with `calendar.add_custom_event`

**Review by jeertmans — CHANGES_REQUESTED** [review]

**Comment by raftar2097** [comment]
Will look into it and get back to you

**Comment by jeertmans** [comment]
Hello @raftar2097, are you still working on this ?

**Pull request closed without merging**